app/Models/WhatToExpect.php

```diff
@@ -17,6 +17,7 @@ class WhatToExpect extends Model implements Sortable
         'icon_type',
         'position',
         'page_id',
+        'landing_page_id',
```

Comment:
Do you need to add a landingPage() function to this class to retrieve the associated landing page?